tests/utils/protocol.test.ts

```diff
@@ -113,6 +113,53 @@ describe('protocol', () => {
     );
   });
 
+  test('migration file should only import used query types', async () => {
+    const packages = await getLocalPackages();
+    const fileName = 'migration_imports_test';
+
+    // Test with only create queries
+    const createQueries = [
+      "create.model({slug: 'model1'})",
+      "create.model({slug: 'model2'})",
+    ];
+    const createProtocol = new Protocol(packages, createQueries);
+
+    // Mock fs.mkdirSync and fs.writeFileSync
```

Comment:
```suggestion
    // Mock `fs.mkdirSync` and `fs.writeFileSync`
```

We usually wrap code in backticks.